conda/recipes/pyraft/meta.yaml

```
@@ -0,0 +1,61 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+# Usage:
+  #   conda build . -c conda-forge -c numba -c rapidsai -c pytorch
+  {% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+  {% set minor_version =  version.split('.')[0] + '.' + version.split('.')[1] %}
+  {% set cuda_version='.'.join(environ.get('CUDA', 'unknown').split('.')[:2]) %}
+  {% set cuda_major=cuda_version.split('.')[0] %}
+  {% set py_version=environ.get('CONDA_PY', 36) %}
+  {% set ucx_py_version=environ.get('UCX_PY_VERSION') %}
+
+package:
+  name: pyraft
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: cuda{{ cuda_major }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script_env:
+    - CC
+    - CXX
+    - VERSION_SUFFIX
+
+requirements:
+  build:
+    - python x.x
+    - setuptools
+    - cython>=0.29,<0.30
+    - cmake>=3.20.1
+    - libraft-core={{ version }}
+    - cudatoolkit {{ cuda_version }}.*
+    - ucx-py {{ ucx_py_version }}
+    - ucx-proc=*=gpu
+  run:
+    - python x.x
+    - dask-cuda {{ minor_version }}
+    - libraft={{ version }}
+    - cupy>=7.8.0,<10.0.0a0
+    - nccl>=2.9.9
+    - ucx-py {{ ucx_py_version }}
+    - ucx-proc=*=gpu
+    - dask>=2021.11.1,<=2021.11.2
```

```suggestion
    - dask>=2021.11.1,<=2022.01.0
```

This is the current pin for 22.02 release